src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  list-style: none;
+  margin: 0;
```

Comment:
1) No margins on BEM blocks
2) This has no effect as you have already reset the margin for all `div`s